doc/conf.py

```diff
@@ -82,7 +82,7 @@
 # The short X.Y version.
 version = nipype.__version__
 # The full version, including alpha/beta/rc tags.
-release = "0.14.0-rc1"
+release = "0.14.0"
```

Comment:
do we want this to be different from `version`?

Reply:
once version is updated, this can stay at `0.14.0`. so just for release this is good, and then post release, we don't have to touch this.